src/rpc/server.cpp

```diff
@@ -231,6 +232,8 @@ static UniValue getrpcinfo(const JSONRPCRequest& request)
     const std::string path = LogInstance().m_file_path.string();
     UniValue log_path(UniValue::VSTR, path);
     result.pushKV("logpath", log_path);
+    UniValue user(UniValue::VSTR, request.authUser);
+    result.pushKV("user", user);
```

Comment:
Yes, this could be made like this. However, I saw in some other (older code) that people were putting string values into UniValue::VSTRs, so I thought, I should follow "best practices". Not that I know, _what_, Bitcoin Core best practices were, but I try to follow already existing code. 

Long story short: yes, I'd prefer putting string directly into result. :)